browser/src/control/Control.UIManager.js

```diff
@@ -1710,12 +1710,19 @@ L.Control.UIManager = L.Control.extend({
 	enableTooltip: function(element) {
 		var elem = $(element);
 		if (window.mode.isDesktop()) {
-			elem.tooltip();
-			elem.on("mousedown", function() {
-				$('.ui-tooltip').fadeOut(function() {
-					$(this).remove();
+			if (this._tooltip) {
+				this._tooltip.tooltip();
+				this._tooltip.tooltip('close');
+				this._tooltip = undefined;
+			}
+			else {
```

Comment:
it was ok before. I meant other else block: 	line 1720

Reply:
mobile version

Reply:
also do we need to do `this._tooltip.tooltip()` before close?

Reply:
Szymon we have to initialize with this._tooltip.tooltip(). Without code doesn't work and throws an error message to debugger console.

It would be  nice handle mobile version in separate patch because I don't know much about mobile side. It can take some time to test on mobile.

Reply:
ok, then please do the less blinking version as you had. Sorry for the noise.